docs/guides/software-lifecycle/security/security-scanning/README.md

```diff
@@ -0,0 +1,208 @@
+
+**Use Cases**:
+- Standardized security reports that enables rapid interchange of scanning tools.
+  - Streamlining management of known security considerations during codebase audits.
+- Discovering security risks in code, such as:
+  - Improper input validation
+  - Weak encryption
+  - Use of dangerous library functions
+  - Other issues that may be difficult to identify via unit testing.
+- Scanning local client repositories to identify exploitable security risks.
+- Implementing a reporting loop in continuous integration (CI) pipelines using GitHub Actions to catch unforeseen risks.
+
+---
+
+## Prerequisites
+To get the most out of `SCRUB`, you'll need:
+
+* Python 3 with the `pip` tool installed
+* Static analysis tools installed and ready for use
+   * CodeQL, SonarQube, and Pylint are some common examples
+* (Optional) Familiarity with BASH and/or Python for potential customizations
+* (Optional) A GitHub repository supporting GitHub Actions
+
+---
+
+## Step-by-Step Guide
+
+SCRUB may be run locally or as a CI workflow action, such as in GitHub Actions. Please see below sections for further details.
+
+### Client-side Scan and Analysis
+The developer's local environment is scanned directly using the `SCRUB` tool. After scanning, a report containing detected security issues is generated. Developers can audit this report for detailed information on detected security concerns.
+
+#### Steps
+1. **Installation**
+   - Install the release version of  [SCRUB](https://nasa.github.io/scrub/installation.html).
+     ```bash
+     pip3 install --upgrade --user nasa-scrub
+     ```
+
+2. **Configuration**
+   - Create a `scrub.cfg` configuration file. This file must be populated with project specific configuration values, depending on the tool that is being used. More information can be found in the [SCRUB documentation](https://nasa.github.io/scrub/configuration.html).
+
+     ``` bash
+     scrub get-conf --output scrub.cfg
+     ```
+
+3. **Scanning**
+   - For more information about running SCRUB, refer to the [user documentation](https://nasa.github.io/scrub/usage.html)
+
+    ```bash
+    scrub run
+    ```
+
+4. **Checking Results**
+   - Review the `<tool>.scrub` file to audit any reported security issues:
+
+      ```bash
+      vi .scrub/<tool>.scrub
+      ```
+
+[View more on advanced SCRUB scan configuration](https://nasa.github.io/scrub/configuration-inputs.html)
+
+> ℹ️ **Note**: Any confirmed security issues should be addressed and mitigated before pushing to remote repositories.
+
+### GitHub.com Actions Analysis on Push and Pull Request
+
+Code is scanned for security risks within the repository. It leverages [GitHub Action](https://github.com/features/actions). The scan is triggered during a push or pull request and any detected security vulnerabilities are reported while blocking merges or pushes to protected branches.
```

Comment:
I think GitHub calls it by the plural: "GitHub Actions".

Reply:
Done.